src/app/api/handle-chapter-request/route.schema.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from "zod";
+
+export const HandleChapterRequest = z.object({
+  chapterRequestId: z.string(),
+  approved: z.boolean(),
+});
+
+export const HandleChapterRequestResponse = z.discriminatedUnion("code", [
+  z.object({
+    code: z.literal("SUCCESS_ACCEPTED"),
```

Comment:
Nit: I think `SUCCESS_ACCEPTED` and `SUCCESS_DECLINED` is strange. Maybe can you use `ACCEPT` / `DECLINE`?

Reply:
As an aside, I think we can merge the cases here. We don't treat accepting and rejecting on the application as different right?